### ISSUE TITLE
`pip install featuretools` breaks conda.cli dependency

### ISSUE THREAD

**Issue by Tagar** [Featuretools/featuretools#240]
### Bug Request Description

`conda` became broken after I installed featuretools package with pip into that a conda envrionment.

```
$ conda create -n py27_rd python==2.7.13
$ source activate py27_rd 
$ pip install featuretools

(py27_rd) $ conda
Traceback (most recent call last):
  File "/opt/conda/envs/py27_rd/bin/conda", line 4, in <module>
    import conda.cli
ImportError: No module named conda.cli
```
Any ideas how to fix this?

**Comment by Seth-Rothschild** [comment]
I was unable to reproduce this issue, and suspect the complaint will be resolved once conda install with `conda install -c featuretools featuretools` is fixed (#246). Closing.

**Comment by Tagar** [comment]
Thanks for looking into this @Seth-Rothschild
